persistence/sql/migrations/sql/20220825134336000001_not_null_constraint_verification_token_flow_id.sqlite3.up.sql

```diff
@@ -0,0 +1,37 @@
+ALTER TABLE identity_verification_tokens
+RENAME TO identity_verification_tokens_;
+
+CREATE TABLE "identity_verification_tokens" (
+"id" TEXT PRIMARY KEY,
+"token" TEXT NOT NULL,
+"used" bool NOT NULL DEFAULT 'false',
+"used_at" DATETIME,
+"expires_at" DATETIME NOT NULL,
+"issued_at" DATETIME NOT NULL,
+"identity_verifiable_address_id" char(36) NOT NULL,
+"selfservice_verification_flow_id" char(36) NOT NULL,
+"created_at" DATETIME NOT NULL,
+"updated_at" DATETIME NOT NULL,
+"nid" char(36),
+FOREIGN KEY (selfservice_verification_flow_id) REFERENCES selfservice_verification_flows (id) ON UPDATE NO ACTION ON DELETE CASCADE,
+FOREIGN KEY (identity_verifiable_address_id) REFERENCES identity_verifiable_addresses (id) ON UPDATE NO ACTION ON DELETE CASCADE
+);
+
+DROP INDEX identity_verification_tokens_id_nid_idx;
+DROP INDEX identity_verification_tokens_nid_id_idx;
+DROP INDEX identity_verification_tokens_token_nid_used_idx;
+DROP INDEX identity_verification_tokens_token_uq_idx;
+DROP INDEX identity_verification_tokens_verifiable_address_id_idx;
+DROP INDEX identity_verification_tokens_verification_flow_id_idx;
+
+CREATE INDEX identity_verification_tokens_id_nid_idx ON identity_verification_tokens (id, nid);
+CREATE INDEX identity_verification_tokens_nid_id_idx ON identity_verification_tokens (nid, id);
+CREATE INDEX identity_verification_tokens_token_nid_used_flow_id_idx ON identity_verification_tokens (nid, token, used, selfservice_verification_flow_id);
```

Comment:
This index has changed for SQLite, but the other migrations did not update this index. Is that intentional and if so why? I see that one query has changed to always use the key, so we probably need to add the missing index for the other databases too?

Reply:
True, missed that - sorry.